### DISCUSSION TITLE
[blocks]: Navbar

### DISCUSSION THREAD

**Discussion by jhgeluk** [shadcn-ui/ui#3252] [Blocks Request]
### Description

A responsive navbar with the ability to handle nested menu items

### Example

https://react-bootstrap.netlify.app/docs/components/navbar

**Comment by abcnow** [comment]
I see some navbars integrated in some of the blocks examples.. but I think to have a block or component with multiple navbars the way you find in https://mui.com/material-ui/react-app-bar/ will be the perfect combination for any project...
